<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Integral_gui</class>
 <widget class="QDialog" name="Integral_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Integral</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mathgraphica.qrc">
    <normaloff>:/images/images/integral.png</normaloff>:/images/images/integral.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_m_factor"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_xmin"/>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_fx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_fx">
     <property name="text">
      <string>f(x)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>= m (quality factor)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_xmax"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../mathgraphica.qrc">:/images/images/integral2.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>d(x)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mathgraphica.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Integral_gui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Integral_gui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
